Doc/library/typing.rst

```diff
@@ -3024,7 +3024,9 @@ Introspection helpers
 
    This is often the same as ``obj.__annotations__``. In addition,
    forward references encoded as string literals are handled by evaluating
-   them in ``globals`` and ``locals`` namespaces. For a class ``C``, return
+   them in ``globals``, ``locals`` and (where applicable)
+   :ref:`type parameter <annotation-scopes>` namespaces.
+   For a class ``C``, return
```

Comment:
we don't usually add `.. versionchanged` notes for bugfixes (and I think this is a bugfix), but I could add one if we think this is significant enough